pkg/http/responses.go

```diff
@@ -49,6 +49,39 @@ type QueryItem struct {
 	Error string          `json:"error,omitempty"`
 }
 
+type ComponentsHealthResponse struct {
+	Results []ComponentHealthItem `json:"results"`
+}
+
+type ComponentHealthItem struct {
+	Component string `json:"componentName"`
+	Type      string `json:"type"`
+	Status    string `json:"status"`
+	ErrorCode string `json:"errorCode,omitempty"`
+	Message   string `json:"message,omitempty"`
+}
+
+type ComponentHealth struct {
```

Comment:
Same question as in protos... Can ComponentHealthItem be used instead of ComponentHealth struct?

Reply:
Used ComponentHealthItem only in http. Similarly, in grpc protos.